include/st_pipeline_api.h

```diff
@@ -319,6 +319,8 @@ struct st_frame {
   /** timing parser meta for st20p_rx_get_frame, only active if
    * ST20P_RX_FLAG_TIMING_PARSER_META */
   struct st20_rx_tp_meta* tp[MTL_SESSION_PORT_MAX];
+  /** TAI timestamp measured right after first packet of the frame was received */
+  uint64_t receive_timestamp;
```

Comment:
If we want to include that, let's include that in every session 

Rest is not applicable, but in st40 and st30 frame modes, this should also be there to minimize the discrepancy between sessions